v0/pipeline/stages/cd/environments-metadata.yaml

```diff
@@ -2,7 +2,10 @@ title: EnvironmentsMetadata
 type: object
 properties:
   parallel:
-    type: boolean
+    oneOf:
+    - type: boolean
+    - type: string
+      pattern: (^<\+input>((\.)((executionInput\(\))|(allowedValues|default|regex)\(.+?\)))*$|^$)     
```

Comment:
@sourabh-awashti can this be execution input also?
Also if only input and execution input, extra check to see it can be allowedValues or not, can we can remove it